src/j1cli.js

```diff
@@ -447,6 +456,15 @@ async function mutateQuestions(j1Client, questions, operation) {
       )}.`
     );
   }
+  if (newFile) {
+    var stringify = JSON.stringify(questions, null, 4);
```

Comment:
could we change this variable name to something like `jsonString`, naming it `stringify` the verb implies its a function or something that can perform an action.

Reply:
Good point